services/helper/mentors.js

```diff
@@ -20,17 +27,49 @@ module.exports = class MentorsHelper {
         })
     }
 
-    static reports(userId) {
-        return new Promise(async (resolve,reject) => {
-            try {
+    static async reports(userId, filterType) {
+        let filterStartDate;
+        let filterEndDate;
+        let totalSessionCreated;
+        let totalsessionHosted;
+        let filters;
+        try {
+            if (filterType === 'MONTHLY') {
+                [filterStartDate, filterEndDate] = utils.getCurrentMonthRange();
+            } else if (filterType === 'WEEKLY') {
+                [filterStartDate, filterEndDate] = utils.getCurrentWeekRange();
+            } else if (filterType === 'QUARTERLY') {
+                [filterStartDate, filterEndDate] = utils.getCurrentQuarterRange();
+            }
 
-                /**
-                 * Your business logic here
-                 */
+            /* totalSessionCreated */ 
+            filters = {
+                createdAt: {
```

Comment:
@aman-g202 what will happen if the session is cancelled ? Should it be included in total session created ?